glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
On Wed, Aug 26, 2015 at 07:31:50AM -0700, Brandon Philips wrote:
> a single bundle is intended to launch a single process at this point.

Is there a reason to avoid allowing multiple application specs inside
a bundle?  Besides the fact that you can do that now (at least with
runC), and that I do this already (with the above-linked IPFS gateway
bundle), we also have @julz sounding positive about this sort of
use-case [1] in the discussion that I've spun off into [2].

[1]: https://groups.google.com/a/opencontainers.org/d/msg/dev/BIxya5eSNLo/gQQ6PHqPFAAJ
[2]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/kPWu-ZavQjs